### PR TITLE
Drop texinfo from build requires.

### PR DESCRIPTION
**Pull request by markolemmetty**
Currently texinfo is in build loop and as we do not really use any of the data generated by it, we should try to drop the deps as much as we can.

### CODE REVIEW

File: rpm/mtools.spec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Buildroot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 Requires(post): /sbin/install-info
 Requires(postun): /sbin/install-info
```
Comment (saukko) [inline rpm/mtools.spec]: Drop these and also %post %preun?

Also drop Group: tags and %clean section.

Comment (markolemmetty) [reply rpm/mtools.spec]: Ok, removed:
Group: Applications/System
Requires(post): /sbin/install-info
Requires(postun): /sbin/install-info
%clean
%post
%preun

File: rpm/mtools.spec [hunk 2]
```diff
@@ -4,14 +4,11 @@ Name: mtools
 Version: 0
 Release: 1
 License: GPLv2+
-Group: Applications/System
 Source: %{name}-%{version}.tar.bz2
 Url: http://mtools.linux.lu/
 Buildroot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
```
Comment (mkosola) [inline rpm/mtools.spec]: delete also buildroot

Comment (markolemmetty) [reply rpm/mtools.spec]: Yes

**Review by mkosola — APPROVED** [review]

**Pull request merged**